README.md

```diff
@@ -9,6 +9,34 @@ FHIR Shorthand (FSH) is a specially-designed language for defining the content o
 
 For more information about the evolving FSH syntax see the [FHIR Shorthand Reference Manual](https://build.fhir.org/ig/HL7/fhir-shorthand/).
 
+## FHIR Foundation Project Statement
+
+### Maintainers
+
+This project is maintained by The MITRE Corporation.
+
+### Issues / Discussion
+
+For SUSHI issues, such as bug reports, comments, suggestions, questions, and feature requests, visit [SUSHI Github Issues](https://github.com/FHIR/sushi/issues).
+
+For discussion of FHIR Shorthand and its associated projects, visit the FHIR Community Chat @ https://chat.fhir.org. The [#shorthand stream](https://chat.fhir.org/#narrow/stream/215610-shorthand) is used for all FHIR Shorthand questions and discussion.
+
+### License
+
+All contributions to this project will be released under the Apache 2.0 License, and a copy of this license can be found [here](LICENSE).
```

Comment:
This is just my preference, but I think I'd tend to use the filename in the link, rather than just "here". So maybe something like: 

```suggestion
All contributions to this project will be released under the Apache 2.0 License, and a copy of this license can be found in [LICENSE](LICENSE).
```

If you agree with this, maybe we can update the links for the Contribution Policy and Security Information to do something similar.